code/modules/economy/requisition/rc_aid.dm

```diff
@@ -106,17 +106,19 @@ ABSTRACT_TYPE(/datum/rc_entry/item/basictool)
 	New()
 		src.name = pick(namevary)
 		src.flavor_desc = "[pick(desc_helpsite)] requires additional supplies [pick(desc_tense)] [pick(desc_crisis)]. [pick(desc_emphasis)]"
-		src.payout += rand(0,40) * 10
+		src.payout += rand(0,60) * 10
+
+		if(prob(60))
+			src.rc_entries += rc_buildentry(/datum/rc_entry/reagent/brutepatch,rand(1,4)*30)
+		if(prob(70))
+			src.rc_entries += rc_buildentry(/datum/rc_entry/reagent/burnpatch,rand(1,4)*30)
```

Comment:
Will this properly find patches inside patch boxes?

Reply:
Not currently, as they're not storage items in the conventional sense. Working on a way to account for this.

Reply:
Now accounted for, including boxes that haven't physically generated their items yet. Not pretty, but functional (tested ingame and should account for any variants of item box).